### ISSUE TITLE
Feature: show version of go-passbolt-cli

### ISSUE THREAD

**Issue by nik-steinebrunner** [passbolt/go-passbolt-cli#41]
Is it possible to add the "--version"/"-v" option, that allows the user to display the current version? This feature would enhance usability and provide users with quick access to essential information about the tool. Thanks :)

**Comment by speatzle** [comment]
Good Idea, i hope i can get around to it this week.

**Comment by kskarthik** [comment]
can be marked as closed

**Comment by speatzle** [comment]
I haven't made a Proper release yet since the current version flag always show the git commit instead of the tag version when build with goreleaser, i am going to fix that SoonTM, make a Release and close this Issue.

**Comment by kskarthik** [comment]
@speatzle I guess we need to dynamically include the `GITHUB_REF` value into the binary

**Comment by kskarthik** [comment]
maybe set the following key in the github actions

```yaml
ldflags:
      - -s -w -X main.build={{.Version}}
      - ./usemsan=-msan

**Comment by speatzle** [comment]
Hi, i just released v0.3.1 with the Version flag.
